cli_tools/gce_windows_upgrade/upgrader/workflows.go

```diff
@@ -174,8 +179,8 @@ func populatePrepareSteps(u *upgrader, w *daisy.Workflow) error {
 
 	// If there isn't an original url, just skip the backup step.
 	if u.originalWindowsStartupScriptURL != nil {
-		fmt.Printf("\nDetected an existing metadata for key '%v', value='%v'. Will backup to '%v'.\n\n", metadataKeyWindowsStartupScriptURL,
-			*u.originalWindowsStartupScriptURL, metadataKeyWindowsStartupScriptURLBackup)
+		fmt.Printf("\nDetected an existing metadata for key '%v', value='%v'. Will backup to '%v'.\n\n", metadataWindowsStartupScriptURL,
+			*u.originalWindowsStartupScriptURL, metadataWindowsStartupScriptURLBackup)
```

Comment:
The order of execution for metadata startup scripts is here: https://cloud.google.com/compute/docs/instances/startup-scripts/windows#order_of_execution_of_windows_startup_scripts

Since you're using windows-startup-script-url, you'll also want to make sure that windows-startup-script-ps1, windows-startup-script-cmd, and windows-startup-script-bat don't have any values set.

Reply:
You are right, that part is missing for both manual guide and automation.
It's a bigger behavior change. Since right now my purpose is to enable the higher version upgrade only for e2e purpose (not open to customers), let's not change this behavior right now.